index.html

```diff
@@ -0,0 +1,195 @@
+                            <h6 class="col-md-12 text-start text-md-center col-3 orange-font fontS-prog-14 mb-0 m-md-2">GDG Sessions</h6>
+                            <p class="col-md-12 col-7 ps-1 ps-md-0 text-start text-md-center fontS-127">
+                                Listen to the speakers from various countries about the messages.
+                            </p>
+                        </li>
+                        <li class="p-3 col-md-2 row m-1 justify-content-center align-items-center">
+                            <img src="img/program_icon_02.png" alt="podium" class="pt-md-2 col-md-auto col-2 program-icon"/>
+                            <h6 class="col-md-12 text-start text-md-center col-3 orange-font fontS-prog-14 mb-0 m-md-2">GDG content</h6>
+                            <p class="col-md-12 col-7 ps-1 ps-md-0 text-start text-md-center fontS-127">
+                                Google technologies in areas like Android, Flutter and so much more!
+                            </p>
+                        </li>
+                        <li class="p-3 col-md-2 row m-1 justify-content-center align-items-center">
+                            <img src="img/program_icon_03.png" alt="podium" class="pt-md-2 col-md-auto col-2 program-icon"/>
+                            <h6 class="col-md-12 text-start text-md-center col-3 orange-font fontS-prog-14 mb-0 m-md-2">GDG Forum</h6>
+                            <p class="col-md-12 col-7 ps-1 ps-md-0 text-start text-md-center fontS-127">
+                                Share your thoughts and opinions with experts for each topic.
+                            </p>
+                        </li>
+                        <li class="p-3 col-md-2 row m-1 justify-content-center align-items-center">
+                            <img src="img/program_icon_04.png" alt="podium" class="pt-md-2 col-md-auto col-2 program-icon"/>
+                            <h6 class="col-md-12 text-start text-md-center col-3 orange-font fontS-prog-14 mb-0 m-md-2">Workshops</h6>
+                            <p class="col-md-12 col-7 ps-1 ps-md-0 text-start text-md-center fontS-127"> 
+                                Full-day workshops, hands-on experiences, and debugging events.
+                            </p>
+                        </li>
+                        <li class="p-3 col-md-2 row m-1 justify-content-center align-items-center">
+                            <img src="img/program_icon_05.png" alt="podium" class="pt-md-2 col-md-auto col-2 program-icon"/>
+                            <h6 class="col-md-12 text-start text-md-center col-3 orange-font fontS-prog-14 mb-0 m-md-2">Networking</h6>
+                            <p class="col-md-12 col-7 ps-1 ps-md-0 text-start text-md-center fontS-127">Get opportunities to network with Coders around the world.</p>
+                        </li>
+                    </ul>
+                </div>
```

Comment:
- [ ] Good job styling this section :raised_hands: . You seem to be missing a border on each card when they are hovered over. KIndly implement this..

**This is the section design on Behance**

![image](https://user-images.githubusercontent.com/86318284/229051094-99e98efc-2901-4751-a068-c01699857950.png)

**Note:** to do this, you need to add an on hover rule on the card.

Reply:
@Nemwel-Boniface Hover rule was added already. Nice suggestion by the way. Thanks

Reply:
Well resolved :heavy_check_mark: